funnel/views/notification.py

```diff
@@ -198,9 +198,12 @@ def tracking_tags(
         :param campaign: Reason for this link being sent (default 'notification' but
             unsubscribe links and other specialized links will want to specify another)
         """
-        tags = {'utm_campaign': campaign, 'utm_medium': transport}
+        tags = {
+            'utm_campaign': f'{self.notification.type}-{self.notification.created_at.strftime("%Y%m%d-%H%M")}',
+            'utm_medium': transport,
+        }
         if not self.notification.for_private_recipient:
-            tags['utm_source'] = self.notification.eventid_b58
+            tags['utm_source'] = 'notification'
```

Comment:
The `if` clause is no longer relevant as there is no event identifier, so nothing that can be traced to the individual.